### PR TITLE
fix unit tests

### PR DESCRIPTION
**Pull request by JanKuczma**

### CODE REVIEW

File: helpers/stable_swap_math/mod.rs [hunk 1]
```diff
@@ -749,7 +751,7 @@ mod tests {
         let token_0_in: u128 = 62463425433;
         let (amount_out, fee_out) = swap_to(0, token_0_in, 1, &reserves, &fees, amp_coef)
             .unwrap_or_else(|_| panic!("Should return SwapResult"));
-        let (amount_in, fee_in) = swap_from(1, amount_out, 0, &reserves, &fees, amp_coef)
+        let (amount_in, fee_in) = swap_from(0, amount_out, 1, &reserves, &fees, amp_coef)
```
Comment (deuszx) [inline helpers/stable_swap_math/mod.rs:754]: Why sudden change?

File: helpers/stable_swap_math/mod.rs [hunk 2]
```diff
@@ -826,54 +798,9 @@ mod tests {
...
-        .unwrap_or_else(|_| panic!("Should work"));
-        assert_eq!(fee_part, 0, "Fee should be 0");
-        assert_eq!(
-            share_by_withdraw_amounts, share,
-            "Share amounts should match"
-        );
-    }
-
-    #[test]
-    fn deposit_liquidity_by_share_and_by_amounts_equality_2() {
```
Comment (deuszx) [inline helpers/stable_swap_math/mod.rs:873]: Why are you removing these tests?

File: helpers/stable_swap_math/mod.rs (hunk 1 above)
Comment (JanKuczma) [reply helpers/stable_swap_math/mod.rs:754]: It didn't matter because the reserves are equal here.

File: helpers/stable_swap_math/mod.rs (hunk 2 above)
Comment (JanKuczma) [reply helpers/stable_swap_math/mod.rs:873]: These tests just show that depending on the method used to calculate liquidity/swap the rounding error influences the result (they were meant to fail).

Comment (deuszx) [reply helpers/stable_swap_math/mod.rs:873]: Failing tests are also good, but if you think they didn't contribute to confidence in the code it's OK to remove.

**Review by deuszx — APPROVED** [review]

**Pull request merged**